Sources/Commands/Options.swift

```diff
@@ -42,4 +42,26 @@ public class ToolOptions {
 }
 
 /// Parser conformance for ColorWrap.
-extension ColorWrap.Mode: StringEnumArgument {}
+extension ColorWrap.Mode: StringEnumArgument {
+    public static var completion: ShellCompletion {
+        return .values([
+            (Auto.description, "auto detect color support"),
+            (Always.description, "always output with color"),
+            (Never.description, "never output with color")
+        ])
+    }
+}
+
+struct PathArgument: ArgumentKind {
```

Comment:
We should add this in a separate PR and with `ArgumentParser` itself, rather than in Commands module.